docsrc/imap/installing.rst

```diff
@@ -440,7 +440,6 @@ Check the output of `groups cyrus` to see what groups it currently belongs to.
 Incorrect groups settings results in saslauthd reporting permission failures::
 
     SASL cannot connect to saslauthd server: Permission denied
-    SASL unable to open Berkeley db /etc/sasldb2: Permission denied
```

Comment:
This line is still valid: it discusses an error from SASL relating to its own usage of Berkeley DB.